packages/by-name/contrast/package.nix

```diff
@@ -5,9 +5,9 @@
 , buildGoModule
 , buildGoTest
 , microsoft
+, runtime ? microsoft
```

Comment:
I don't think we want to make this configurable, but rather embed multiple reference value sets in case in case we support multiple. So passing microsoft should be enough.